server/webhook_test.go

```diff
@@ -0,0 +1,88 @@
+package server
+
+import (
+	"context"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSendToWebhookIntegration(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	s := &Server{
+		StartTime: time.Now(),
```

Comment:
Why do we need this in the test?

Same for below.

Reply:
We are calling `s.sendToWebhook(context.Background(), mattermost.URL, validPayload)`. Is there another way?

Reply:
Sorry, I meant the `StartTime` field. Is that needed?